src/cascadia/TerminalApp/GlobalAppSettings.idl

```diff
@@ -9,6 +9,12 @@ import "TerminalSettings.idl";
 
 namespace TerminalApp
 {
+    struct LaunchPosition
+    {
+        Windows.Foundation.IReference<Int64> X;
+        Windows.Foundation.IReference<Int64> Y;

```

Comment:
I'm completely shocked that this works. I really didn't expect it to work. **who owns these objects?** Passing a struct of COM objects is _very, very unusual_.

Was there something wrong with the sentinel value implementation?

Reply:
Offline discussion:
https://docs.microsoft.com/en-us/uwp/midl-3/intro#types
> A struct type defines a Windows Runtime structure that contains data members. Structs are value types, and they do not require heap allocation. A data member of a struct type must either be a value type or a nullable type. Struct types do not support inheritance.

Turns out this is a feature of MIDL 3. `IReference` is a special case that can be included in a struct.